internal/services/redis/redis_cache_resource.go

```diff
@@ -145,6 +145,10 @@ func resourceRedisCache() *pluginsdk.Resource {
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
+						"entra_enabled": {
```

Comment:
For continuity, can we name this `active_directory_authentication_enabled` (or `active_directory_enabled` if this is for something other than authentication)?

Reply:
FWIW, since MSFT is rebranding AAD to entra - this was exactly my thought behind the naming :) Still wanna go with AD?

Reply:
Yeah we'll tackle that later as part of a larger effort, for now it's better to keep it consistent and stick with the Azure Active Directory branding 👍